factory/api/apiResolverFactory.go

```diff
@@ -451,7 +456,7 @@ func createArgsSCQueryService(args *scQueryElementArgs) (*smartContract.ArgsNewS
 			return nil, nil, err
 		}
 
-		argsHook.Accounts, err = createNewAccountsAdapterApi(args, argsHook.BlockChain)
+		argsHook.Accounts, storageManager, err = createNewAccountsAdapterApi(args, argsHook.BlockChain)
```

Comment:
`Accounts`, `BlockChain` already have value in `argHooks `.
`storageManager` can be defined on line 395